### PR TITLE
Generate stable test outputs for better bundle-size comparisions

### PR DESCRIPTION
**Pull request by jridgewell**
I'm getting annoyed with all the `±0.01kb` changes in our bundle-size outputs, which aren't actually cased by any change in the file. It's just Terser detecting the relative frequency of chars in the files it outputs, which it uses to prioritize certain chars over others.

### CODE REVIEW

File: build-system/pr-check/bundle-size-module-build.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const jobName = 'bundle-size-module-build.js';
  * Steps to run during push builds.
  */
 function pushBuildWorkflow() {
-  timedExecOrDie('amp dist --noconfig --esm');
+  timedExecOrDie('amp dist --noconfig --esm --version_override 0000000000000');
```
Comment (estherkim) [inline build-system/pr-check/bundle-size-module-build.js:21]: wait so different amp versions caused different bundle sizes? lol

**Review by estherkim — APPROVED** [review]

File: build-system/pr-check/bundle-size-module-build.js (hunk 1 above)
Comment (jridgewell) [reply build-system/pr-check/bundle-size-module-build.js:21]: Brotli compression could find different substrings to backreference. Having a constant `0000…` means everything will compare to same bolierplate between changes.

Comment (rsimha) [reply build-system/pr-check/bundle-size-module-build.js:21]: Reminds me of this conversation from a couple years ago when we first adopted brotli: https://github.com/ampproject/amphtml/pull/24418#issuecomment-530156261. Glad it's being fixed here. 😃

**Review by rsimha — APPROVED** [review]

File: build-system/tasks/helpers.js [hunk 2]
```diff
@@ -542,6 +542,22 @@ async function minify(code, map) {
...
+      // eslint-disable-next-line local/camelcase
+      nth_identifier: {
+        get(num) {
+          const leading =
+            '$ABCDEFGHIJKLMNOPQRSTUVWXYZ_abcdefghijklmnopqrstuvwxyz';
+          const all = leading + '0123456789';
+          let charset = leading;
+          let id = '';
+          do {
+            id += charset[num % charset.length];
```
Comment (samouri) [inline build-system/tasks/helpers.js:554]: The first sequence of chars goes as I'd expect. The second round I found surprising:
1. A
2. B
3. C
4. ...

55: 'AA'
56: 'BA'
57: 'CA'

Is that intentional? I would have expected AA, AB, AC

Comment (jridgewell) [reply build-system/tasks/helpers.js:554]: It's the easiest way to handle it, but I don't believe it makes a difference.

Comment (samouri) [reply build-system/tasks/helpers.js:554]: In that case, LGTM

File: build-system/pr-check/bundle-size-module-build.js (hunk 1 above)
Comment (danielrozenberg) [reply build-system/pr-check/bundle-size-module-build.js:21]: Oops, definitely my bad for missing that bit when I spinned off the Prod build jobs! Thanks for fixing this Justin

**Pull request merged**